src/freenet/pluginmanager/PluginReplySenderFCP.java

```diff
@@ -30,8 +33,7 @@ public void send(SimpleFieldSet params, Bucket bucket) throws PluginNotFoundExce
 		// like in linux everthing is a file, in Plugintalker everything is a plugin. So it throws PluginNotFoundException
 		// instead fcp connection errors 
 		if (handler.isClosed()) throw new PluginNotFoundException("FCP connection closed");
-		FCPPluginReply reply = new FCPPluginReply(pluginname, clientSideIdentifier, params, bucket);
+        FCPPluginServerMessage reply = new FCPPluginServerMessage(pluginname, clientSideIdentifier, params, bucket);
```

Comment:
The whole class which the above changes is deprecated - it is the legacy plugin-fcp API which exists so old plugins keep working. Thus it uses the deprecated constructor which was part of it.
-> No change required.